py3status/modules/xrandr.py

```diff
@@ -138,13 +136,8 @@ def _get_layout(self):
             'disconnected': OrderedDict()
         })
 
-        current = Popen(['xrandr'], stdout=PIPE)
-        for line in current.stdout.readlines():
-            try:
-                # python3
-                line = line.decode()
-            except:
-                pass
+        current = self.py3.command_output('xrandr')
+        for line in current.splitlines(True):
```

Comment:
It seems silly to carry unneeded cruft just removing the `True` would be a good start